### PR TITLE
chromedriver: Switch to Chromium's upstream-info.json

### LINKED ISSUE #85629: incompatible chrome and chromedriver
**Describe the bug**
Chrome and Chromedriver are incompatible on my install.

**To Reproduce**
1. Install chrome and chromedriver from nixpkgs
2. Try using chrome with selenium.
3. Get this error
    This version of ChromeDriver only supports Chrome version 78

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.108, NixOS, 19.09.2218.a81842b6994 (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.3`
 - channels(root): `"nixos-19.09.2218.a81842b6994"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`


```

### PR DESCRIPTION
**Pull request by primeos**
This enables automatic updates and fixes #85629.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/browsers/chromium/update.py [hunk 1]
```diff
@@ -84,7 +98,10 @@ def get_channel_dependencies(channel):
             # the next one.
             continue
 
-        channel['deps'] = get_channel_dependencies(channel)
+        channel['deps'] = get_channel_dependencies(channel, channel_name)
+        if channel_name == 'stable':
+            channel['chromedriver'] = get_matching_chromedriver(channel['version'])
+            print(channel['chromedriver'])
```
Comment (lukegb) [inline pkgs/applications/networking/browsers/chromium/update.py:104]: This print statement is probably confusing out of context; consider dropping it?

**Review by lukegb — APPROVED** [review]
LG, other than that nit.

Finagled the upstream-info.json to force it to rewrite all the entries and it output the exact same thing in this commit, so LGTM.

**Review by alexanderkranga — APPROVED** [review]
https://github.com/NixOS/nixpkgs/issues/85629#issuecomment-734753118

File: pkgs/applications/networking/browsers/chromium/update.py (hunk 1 above)
Comment (primeos) [reply pkgs/applications/networking/browsers/chromium/update.py]: Nice catch, thanks! I only had that for debugging purposes and missed it when reviewing the final diff.

**Pull request merged**